The Wazuh Docker image cannot be built during the Nightly

## Description

While analyzing https://github.com/wazuh/wazuh-qa-automation/issues/6781, it has been detected that the Docker image has not been built due to the following failure in the `build-and-push (wazuh-dashboard)` stage:

```
#16 [linux/arm64 stage-1 4/7] RUN yum install shadow-utils -y &&     yum clean all &&     getent group wazuh-dashboard || groupadd -r -g 1000 wazuh-dashboard &&     useradd --system             --uid 1000             --no-create-home             --home-dir /usr/share/wazuh-dashboard             --gid wazuh-dashboard             --shell /sbin/nologin             --comment "wazuh-dashboard user"             wazuh-dashboard &&     chmod 700 /entrypoint.sh &&     chmod 700 /wazuh_dashboard_config.sh &&     mkdir -p /usr/share/wazuh-dashboard &&     chown 1000:1000 /usr/share/wazuh-dashboard &&     chown 1000:1000 /*.sh &&     mkdir -p /usr/share/wazuh-dashboard/plugins/wazuh/public/assets/custom
#16 CANCELED
------
 > [linux/amd64 builder 7/7] RUN setcap 'cap_net_bind_service=-ep' /usr/share/wazuh-dashboard/node/fallback/bin/node:
0.050 Failed to set capabilities on file 'setcap': No such file or directory
------
Dockerfile:27
--------------------
  25 |     RUN mkdir -p $INSTALL_DIR/data/wazuh/logs && chmod -R 775 $INSTALL_DIR/data/wazuh/logs
  26 |     RUN setcap 'cap_net_bind_service=-ep' /usr/share/wazuh-dashboard/node/bin/node
  27 | >>> RUN setcap 'cap_net_bind_service=-ep' /usr/share/wazuh-dashboard/node/fallback/bin/node
  28 |     
  29 |     ################################################################################
--------------------
ERROR: failed to build: failed to solve: process "/bin/sh -c setcap 'cap_net_bind_service=-ep' /usr/share/wazuh-dashboard/node/fallback/bin/node" did not complete successfully: exit code: 1
Error: Process completed with exit code 1.
```

The build has been relaunched to check if it was a temporary failure, but it failed again.

## Evidence

- Build: https://github.com/wazuh/wazuh-docker/actions/runs/24869639955/attempts/1
- Rebuild: https://github.com/wazuh/wazuh-docker/actions/runs/24869639955

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -49,6 +49,7 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Delete setcap command on deprecated file ([#2345](https://github.com/wazuh/wazuh-docker/pull/2345))
 - Modify the choice of a correct tag ([#2313](https://github.com/wazuh/wazuh-docker/pull/2313))
 - Artifact URL download fix ([#2306](https://github.com/wazuh/wazuh-docker/pull/2306))
 - Change API query method. ([#2275](https://github.com/wazuh/wazuh-docker/pull/2275))

--- a/build-docker-images/wazuh-dashboard/Dockerfile
+++ b/build-docker-images/wazuh-dashboard/Dockerfile
@@ -24,7 +24,6 @@ RUN mkdir -p $INSTALL_DIR/data/wazuh && chmod -R 775 $INSTALL_DIR/data/wazuh
 RUN mkdir -p $INSTALL_DIR/data/wazuh/config && chmod -R 775 $INSTALL_DIR/data/wazuh/config
 RUN mkdir -p $INSTALL_DIR/data/wazuh/logs && chmod -R 775 $INSTALL_DIR/data/wazuh/logs
 RUN setcap 'cap_net_bind_service=-ep' /usr/share/wazuh-dashboard/node/bin/node
-RUN setcap 'cap_net_bind_service=-ep' /usr/share/wazuh-dashboard/node/fallback/bin/node
 
 ################################################################################
 # Build stage 1 (the current Wazuh dashboard image):
@@ -63,13 +62,13 @@ RUN yum install shadow-utils -y && \
     yum clean all && \
     getent group $GROUP || groupadd -r -g 1000 $GROUP && \
     useradd --system \
-            --uid 1000 \
-            --no-create-home \
-            --home-dir $INSTALL_DIR \
-            --gid $GROUP \
-            --shell /sbin/nologin \
-            --comment "$USER user" \
-            $USER && \
+    --uid 1000 \
+    --no-create-home \
+    --home-dir $INSTALL_DIR \
+    --gid $GROUP \
+    --shell /sbin/nologin \
+    --comment "$USER user" \
+    $USER && \
     chmod 700 /entrypoint.sh && \
     chmod 700 /wazuh_dashboard_config.sh && \
     mkdir -p $INSTALL_DIR && \

